content/id/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -0,0 +1,196 @@
+
+{{< note >}}
+Fitur ini hanya tersedia untuk penyedia _cloud_ atau lingkungan yang mendukung _load balancer_ eksternal.
+{{< /note >}}
+
+Ketika membuat servis, kamu mempunyai opsi untuk membuat jaringan _cloud load balancer_ secara otomatis.
+Hal ini menyediakan akses eksternal alamat IP yang dapat mengirim lalu lintas melalui porta yang tepat pada klaster _node_ kamu
+_asalkan klaster kamu beroperasi pada lingkungan yang mendukung dan terkonfigurasi dengan paket penyedia cloud load balancer yang benar_.
+
+Untuk informasi mengenai penyediaan dan penggunaan sumber daya _Ingress_ yang dapat memberikan
+servis URL yang dapat dijangkau secara eksternal, penyeimbang beban lalu lintas, terminasi SSL, dll.,
+silahkan cek dokumentasi [_Ingress_](/docs/concepts/services-networking/ingress/)
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Berkas konfigurasi
+
+Untuk membuat _load balancer_ eksternal, tambahkan baris dibawah ini ke
+[berkas konfigurasi servis](/docs/concepts/services-networking/service/#loadbalancer) kamu:
+
+```yaml
+    type: LoadBalancer
+```
+
+Berkas konfigurasi kamu mungkin terlihat seperti ini:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-service
+spec:
+  selector:
+    app: example
+  ports:
+    - port: 8765
+      targetPort: 9376
+  type: LoadBalancer
+```
+
+## Menggunakan kubectl
+
+Kamu dapat membuat servis dengan perintah `kubectl expose` dan
+_flag_ `--type=LoadBalancer`:
+
+```bash
+kubectl expose rc example --port=8765 --target-port=9376 \
+        --name=example-service --type=LoadBalancer
+```
+
+Perintah ini membuat servis baru dengan menggunakan pemilih yang sama dengan
+sumber daya yang dirujuk (dalam hal contoh diatas, pengendali replikasi bernama `example`).
+
+Untuk informasi lebih lanjut, termasuk opsi _flag_, mengacu kepada
+[referensi `kubectl expose`](/docs/reference/generated/kubectl/kubectl-commands/#expose).
+
+## Menemukan alamat IP kamu
+
+Kamu dapat menemukan alamat IP yang telah dibuat untuk servis kamu dengan mendapatkan
+informasi servis melalui `kubectl`:
+
+```bash
+kubectl describe services example-service
+```
+
+yang seharusnya menghasilkan keluaran seperti ini:
+
+```bash
+    Name:                   example-service
+    Namespace:              default
+    Labels:                 <none>
+    Annotations:            <none>
+    Selector:               app=example
+    Type:                   LoadBalancer
+    IP:                     10.67.252.103
+    LoadBalancer Ingress:   192.0.2.89
+    Port:                   <unnamed> 80/TCP
+    NodePort:               <unnamed> 32445/TCP
+    Endpoints:              10.64.0.4:80,10.64.1.5:80,10.64.2.4:80
+    Session Affinity:       None
+    Events:                 <none>
+```
+
+Alamat IP tercantum di sebelah `LoadBalancer Ingress`.
```

Comment:
👍 `LoadBalancer Ingress` is good way of writing with backtick, because it's referring to fields in above example